web_service.go

```diff
@@ -118,7 +118,7 @@ func (w *WebService) QueryParameter(name, description string) *Parameter {
 // QueryParameter creates a new Parameter of kind Query for documentation purposes.
 // It is initialized as not required with string as its DataType.
 func QueryParameter(name, description string) *Parameter {
-	p := &Parameter{&ParameterData{Name: name, Description: description, Required: false, DataType: "string"}}
+	p := &Parameter{&ParameterData{Name: name, Description: description, Required: false, DataType: "string", CollectionFormat: CollectionFormatMulti.String()}}
```

Comment:
I added a default CollectionFormat here only because it has no effect if AllowMultiple is not set to true.  The alternative is to handle the existence of a non-empty string value in the build_path.go of go-restful-openapi.

Reply:
The [spec](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md) says ```Default value is csv``` so maybe we should use this instead.

Reply:
Ah, good call.  I will change that.  Do you prefer a new squashed CR, or are you ok with multiple commits?

Reply:
squash it yes please, i will merge right after.